### DISCUSSION TITLE
typst output format?

### DISCUSSION THREAD

**Discussion by aghaynes** [rstudio/gt#1584] [Ideas]
quarto has recently included typst as an output format. Tables generated with gt lack their usual panache though... are there any plans to extend gt to typst tables? Considering how fast it seems to be in comparison to latex... it's an attractive format (for me at least)...

![image](https://github.com/rstudio/gt/assets/6594028/4543c122-9616-40d8-a13b-f5c8094a69ed)

**Comment by olivroy** [comment]
See [#1500](https://github.com/rstudio/gt/issues/1500#issuecomment-2209180130). Feel free to give feedback if you have some. This should work natively with Quarto > 1.5. https://quarto.org/docs/blog/posts/2024-07-02-beautiful-tables-in-typst/

There is still an overflow issue, but it looks better.
![image](https://github.com/rstudio/gt/assets/52606734/15c6b02c-89cc-42e0-9228-74f4b00d49a1)

**Reply by olivroy** [reply]
I opened https://github.com/quarto-dev/quarto-cli/issues/10228 about text overflow Apparently, this is a typst issue and may or may not resolve over time! (neither gt or Quarto's faults)
